prisma/seed/users.ts

```diff
@@ -15,14 +16,30 @@ export const populateUsers = async () => {
             emailVerified: true,
             firstName: "Jessica",
             lastName: "Williamson",
-            avatar: "https://gravatar.com/avatar/3bfaef00e02a22f99e17c66e7a9fdd31?s=400&d=robohash&r=x",
+            avatar: generateGravatarUrl("jessica.williamson@gmail.com"),
             timezone: "Australia/Melbourne",
             countryCode: "AU",
             gender: {
                 connect: {
                     abbreviation: "F",
                 },
             },
+            oAuthProfiles: {
+                create: {
+                    providerUserId: "1234567",
+                    providerUsername: "jessica-discord",
+                    provider: {
+                        connectOrCreate: {
+                            where: {
```

Comment:
I think the `where` clause in the connectOrCreate statement should refer to the `id` instead of the `name` to avoid the type mismatch.
I found a stackoverflow post about where the OP was having the same issue. 
[https://stackoverflow.com/questions/68874214/how-to-use-connectorcreate-with-many-to-many-in-prisma](https://stackoverflow.com/questions/68874214/how-to-use-connectorcreate-with-many-to-many-in-prisma)

Reply:
I remember that now I had this issue and that was fixed by marking OAuthProvider name @unique, it couldn't connect because it wasn't unique, connect only works if the field used to connect is unique. 

I don't know why it's not in the schema, could be because I've switched branches so many times and it got lost 

```
model OAuthProvider {
  id   Int    @id @default(autoincrement())
  name String @unique

  createdAt DateTime @default(now()) @db.Timestamptz()
  updatedAt DateTime @updatedAt

  userOAuthProfiles UserOAuthProfile[]
}
```

Reply:
I have pull the latest changes from the remote branch and it's still has type mismatch errors. I checked my 0authProvider model and it's correct. I'm not sure what else to do.